web/src/components/LandingPageSlider.tsx

```diff
@@ -0,0 +1,176 @@
+import { LandingPageActionTile } from "@/components/LandingPageActionTile";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { QUERIES, ROUTES, routeWithQuery } from "@/lib/domain-logic/routes";
+import analytics from "@/lib/utils/analytics";
+import { useRouter } from "next/router";
+import { ReactElement, useRef, useState } from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick-theme.css";
+import "slick-carousel/slick/slick.css";
+import { useMountEffect } from "../lib/utils/helpers";
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+function SampleNextArrow(props: any) {
+  const { onClick } = props;
+  return (
+    <button onClick={onClick} className="slider-button next">
+      {">"}
+    </button>
+  );
+}
+function SamplePrevArrow(props: any) {
+  const { onClick } = props;
+  return (
+    <button onClick={onClick} className="slider-button prev">
+      {"<"}
+    </button>
+  );
+}
+
+export const LandingPageSlider = (): ReactElement => {
+  const router = useRouter();
+  const { Config } = useConfig();
+  const [activeSlide, setActiveSlider] = useState(0);
+  const [slideBreakPoint, setSlideBreakPoint] = useState(7);
+  const cardRef = useRef<null | HTMLButtonElement>(null);
+  const sliderRef = useRef<null | HTMLDivElement>(null);
+
+  const getNumberOfTilesToShow = (): number => {
+    const sliderWidth = sliderRef.current?.clientWidth;
+    const cardWidth = cardRef.current?.parentElement?.clientWidth;
```

Comment:
I'm confused by this line - if you want the width of the parent element, why not just put the ref on the parent element instead of selecting the parent here? This is potentially risky code because if someone say, adds another wrapping div around that ref, the "parent element" would change and then this code would break